eta/core/types.py

```diff
@@ -489,6 +489,42 @@ def is_valid_path(path):
         )
 
 
+class NpzFile(File, ConcreteData):
```

Comment:
@jeffreydominic note that I made this `NpzFile` instead of `NPZFile`, to match our existing conventions like `NpzModelWeights`. So you'll need to update the associated usages in THETA.

Sorry for the inconvenience.

Reply:
That's fine. Should be any easy fix. @kunyilu It'll be good if you could add this fix to your `types.py` to avoid merge conflicts.